proxy/src/main/java/com/wavefront/agent/handlers/ReportPointHandlerImpl.java

```diff
@@ -67,6 +69,8 @@
     MetricsRegistry registry = setupMetrics ? Metrics.defaultRegistry() : LOCAL_REGISTRY;
     this.receivedPointLag = registry.newHistogram(new MetricName(handlerKey.toString() +
         ".received", "", "lag"), false);
+    this.pointTagCount = registry.newHistogram(new MetricName(handlerKey.toString(), "", "pointTagCount"), false);
```

Comment:
rename this to 
pointTagCount.received

Reply:
probable points.tagCount.received.

Reply:
feel `points.tagCount.received` is slightly better

Reply:
I have to disagree with both @srujann and @sushantdewan123 :) 
We should try to keep our naming at least somewhat consistent - I suggest naming it `.received.tagCount`, similar to `.received.lag` right above it. The final metric names would be like `~proxy.points.2878.received.tagCount.p99` etc.

Reply:
@basilisk487 the only difference here is that this metric is recorded pre-validation.
Hence I put in the keyword `raw` in there :)

Reply:
Are you okay with :)
**~proxy.points.2878.received.raw.tagCount.p99**

Reply:
what else would go under `received.raw.` metric tree? i would imagine `.received.tagCount` would be pre-validation, and if we need to, we could add `.sent.tagCount` for post-validation?